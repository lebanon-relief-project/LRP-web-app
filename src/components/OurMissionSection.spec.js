import React from "react";
import { render } from "@testing-library/react";
import OurMission from "./OurMissionSection";
import "jest-styled-components";

describe("the OurMission component", () => {
  let container;
  it("should match the snapshot", () => {
    ({ container } = renderOurMissionSection());

    expect(container).toMatchSnapshot();
  });
});

const renderOurMissionSection = () => {
  return render(<OurMission />);
};
