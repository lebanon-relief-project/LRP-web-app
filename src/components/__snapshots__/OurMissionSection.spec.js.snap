// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the OurMission component should match the snapshot 1`] = `
<div>
  <section
    class="sc-gtssRu jGobNw"
  >
    <div
      class="sc-bdnylx icjZAh"
    >
      <h2
        class="sc-dlnjPT iGXfwf"
      >
        Our mission
      </h2>
      <p
        class="sc-hKFyIo hqFVHM"
      >
        Lorem lobortis aliquam leo nisi vitae egestas elit. Tellus ultrices et velit luctus. Justo, in mauris pulvinar cursus velit at sollicitudin. Eget pretium nunc placerat sed at eget est. Exercitation veniam consequat sunt nostrud amet. Pretium nunc placerat sed at eget est. Exercitation veniam consequat sunt nostrud amet.
      </p>
    </div>
  </section>
</div>
`;
